shared/src/vyxal/Elements.scala

```diff
@@ -193,6 +194,11 @@ object Elements:
           scribe.warn(s"Could not invert matrix $l")
           l
         }
+      case l: VList => l.vmap({
+          case n: VNum => VNum(NumberHelpers.toBinary(n).size)
+          case default => default
+        })
```

Comment:
```suggestion
      case l: VList => l.vmap {
          case n: VNum => VNum(NumberHelpers.toBinary(n).size)
          case default => default
        }
```
You can actually omit the parentheses. If you only have one argument to pass, Scala lets you use `{}` instead of `({})`.